core/templates/css/oppia.css

```diff
@@ -376,7 +376,7 @@ legend {
 }
 
 .oppia-dashboard-container .exp-private-text {
-  color: #aaa;
```

Comment:
Original color of text:
![Screen Shot 2021-04-09 at 3 43 49 PM](https://user-images.githubusercontent.com/35385366/114247667-80a72780-994a-11eb-81eb-2d8a40d49d1b.png)